content/en/docs/setup/upgrade/in-place/index.md

```diff
@@ -59,8 +59,7 @@ can be found in the `bin/` subdirectory of the downloaded package.
     then you must provide the same `-f` flag value to the `istioctl upgrade` command.
     {{< /warning >}}
 
-    `istioctl upgrade` does not support the `--set` flag. Therefore, if you
-    installed Istio using the `--set` command, create a configuration file with
+    If you installed Istio using the `--set` flag, create a configuration file with
```

Comment:
Why do we need to do this, if `istioctl upgrade` does now support `--set`?

cc @ostromart

Reply:
good point - this is not the right way to word this. i propose something like:
```
If you installed Istio using `--set` flags, ensure that you pass the same `--set` flags to upgrade, otherwise the customizations done with `--set` will be reverted. For production use, the use of a configuration file instead of `--set` is recommended.
```

Reply:
Thanks @ostromart